src/mocks/data/draft_reports_object.json

```diff
@@ -0,0 +1,50 @@
+{
+  "status": "processing", // enum: one of ["processing", "failed", "delivered"]
+  "current_stage": "dex-upload",
+  "current_action": "metadata-validation",
+  "file_name": "test.txt",
+  "transport_id": "unique_guid", //represents the id for the entire transaction i.e. upload_id
```

Comment:
we're all familiar with upload_id - why not call it that instead of having a new name for it? 
I'm sure FHIR could use upload_id for their submisions as well

Reply:
Good point, the pros and cons should be weighed in regards to familiarity vs clarity.